### DISCUSSION TITLE
.Net appliaction Monitoring in Skywalking Failing.

### DISCUSSION THREAD

**Discussion by Maurya243** [apache/skywalking#11879] [Q&A]
Hi Team,
OAP version : 9.7
.Net agent version : 2.1.0

We are unable to monitor .Net application using skywalking. We have followed the procedure from the below link and getting error as follows : 
"SkyApm.Transport.Grpc.V8.ServiceRegister : Report service instance properties fail.
Grpc.Core.RpcException: Status(StatusCode="Unimplemented", Detail="Method not found: skywalking.v3.ManagementService/reportInstanceProperties")
   at SkyApm.Transport.Grpc.V8.ServiceRegister.<>c__DisplayClass9_0.<<ReportInstancePropertiesAsync>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at SkyApm.Transport.Grpc.Common.Call.Execute[T](Func`1 task, Func`1 errCallback, Func`1 errMessage)
2024-02-14 15:36:34.567 +05:30 [sample_app] [Warning] SkyApm.Transport.Grpc.ConnectionManager : Connection state changed. Status(StatusCode="Unimplemented", Detail="Method not found: skywalking.v3.ManagementService/reportInstanceProperties")"

https://github.com/SkyAPM/SkyAPM-dotnet

Thanks in advance

**Comment by wu-sheng** [comment]
Hi, that project is out of the Apache, it is better you submit issues on that repository directly.

**Reply by Maurya243** [reply]
Thank you for prompt response.

Does 9.7 skywalking support .Net agent? We had tried with 9.4 skywalking and which was working fine., able to see data as well.
Regarding the error log we will submit an issue on the respective repository.

**Reply by wu-sheng** [reply]
In theory, it is. Backend is protocol oriented, which is very stable.

**Reply by wu-sheng** [reply]
9.7 and 9.4 should not have significant difference AFAIK.

**Comment by wu-sheng** [comment]
> Detail="Method not found: skywalking.v3.ManagementService/reportInstanceProperties"


About this, this is a service existing for a while, why 9.7 reports not found?
